ctgan/synthesizers/base.py

```diff
@@ -57,13 +57,38 @@ def wrapper(self, *args, **kwargs):
 
 
 class BaseSynthesizer:
-    """Base class for all default synthesizers of ``CTGAN``.
-
-    This should contain the save/load methods.
-    """
+    """Base class for all default synthesizers of ``CTGAN``."""
 
     random_states = None
 
+    def __getstate__(self):
+        """Remove the ``torch`` generator when pickling."""
```

Comment:
Can we be slightly more descriptive? ie. replace torch generator with state or something like that?